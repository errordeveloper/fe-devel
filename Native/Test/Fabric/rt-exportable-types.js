FC = require("Fabric").createClient();

var node = FC.DG.createNode("foo");
try {
  node.addMember("bar", "Data");
  FC.flush();
}
catch (e) {
  console.log("node.addMember: " + e);
}

FC.close();
