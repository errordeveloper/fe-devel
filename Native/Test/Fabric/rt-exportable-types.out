node.getData('bar'): Fabric core exception: DG.foo.getData('{"memberName":"bar","sliceIndex"...'): unable to convert ValueProducer to JSON
